packages/store/mud.config.ts

```diff
@@ -0,0 +1,43 @@
+import { MUDConfig, ContractSchemaType } from "@latticexyz/cli";
+
+const config: MUDConfig = {
```

Comment:
I think this config interface makes sense! We should probably have a `StoreConfig` that includes the table definitions, and then extend the type to a `WorldConfig` when used with the World framework. In the latter we could add a field for `route` in each `Table`, and a separate `systems` top level key to define the systems in a similar fashion.